src/openrct2/object/ObjectRepository.cpp

```diff
@@ -68,13 +89,14 @@ struct ObjectEntryEqual
     }
 };
 
+using ObjectIdentifierMap = std::unordered_map<std::string, size_t, ObjectIdentifierHash, ObjectIdentifierEqual>;
```

Comment:
Any particular reason you are using ObjectIdentifierHash isntead of std::hash<std::string>?

Reply:
Lack of C++ skills.